olwebsite-app/src/components/SearchBox/SearchBox.jsx

```diff
@@ -0,0 +1,49 @@
+import React, {useState} from "react";
+import PropTypes from "prop-types";
+import { FaSearch } from "react-icons/fa";
+
+const SearchBox = ({ onSearch, placeholder }) => {
+
+  // If we decide to search on change, raise the state
+  const [searchQuery, setSearchQuery] = useState("");
+
+  const handleKeyPress = key => {
+      if(key === "Enter") {
+          onSearch(searchQuery);
+      }
+  }
+
+  /************************************
+   * Render
+   ************************************/
+
+  return (
+    <div className="search-box">
+      <div className="search-box-input-wrapper">
+        <FaSearch 
+          className="search-box-icon" 
+          onClick={() => onSearch(searchQuery)}
+        />
+        <input
```

Comment:
Should have an on focus attribute for this input box to give a more reactive/responsive UI